test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionDescriptorProviderTests.cs

```diff
@@ -38,6 +38,20 @@ public void GetDescriptors_GetsDescriptorsOnlyForValidActions()
         }
 
         [Fact]
+        public void GetDescriptors_DisplayNameIncludesAssemblyQualifiedTypeReference()
+        {
+            // Arrange
+            var provider = GetProvider(typeof(PersonController).GetTypeInfo());
+
+            // Act
+            var descriptors = provider.GetDescriptors();
+            var descriptor = descriptors.Single(ad => ad.Name == nameof(PersonController.GetPerson));
+
+            // Assert
+            Assert.Equal($"Microsoft.AspNetCore.Mvc.Internal.ControllerActionDescriptorProviderTests+PersonController, Microsoft.AspNetCore.Mvc.Core.Test, Version=1.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60.GetPerson", descriptor.DisplayName);
+        }
```

Comment:
@rynowak This is how  the method name would look with the AssemblyQualifiedName, I find it a bit ugly for error messages, maybe we could do something like "GetPerson - (Microsoft.AspNetCore.Mvc.Internal.ControllerActionDescriptorProviderTests+PersonController, Microsoft.AspNetCore.Mvc.Core.Test, Version=1.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60)"